specification/gp-connect-user-permissions.yaml

```diff
@@ -220,8 +225,19 @@ components:
           items:
             type: object
             required:
+              - severity
               - code
+              - details
```

Comment:
Looking at the info in Confluence and the signing service error I don't think `details` is required.